meshtastic/version.py

```
@@ -1,13 +1,18 @@
-"""Version lookup utilities, isolated for cleanliness"""
+"""Version lookup utilities, isolated for cleanliness."""
+
 import sys
+
 try:
     from importlib.metadata import version
 except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)